Makefile

```diff
@@ -48,4 +48,18 @@ constd/docker: constd
 constd/metal: constd
 	CONSTD_EXEC_MODE=metal .bin/constd $(PWD)/constd/example-project/runnables.wasm.zip
 
-.PHONY: sat constd
\ No newline at end of file
+constd/metal/otel/hc: constd
+	ATMO_TRACER_TYPE="honeycomb" \
+	ATMO_TRACER_PROBABILITY=${ATMO_TRACER_PROBABILITY} \
+	ATMO_TRACER_HONEYCOMB_ENDPOINT=${ATMO_TRACER_HONEYCOMB_API_KEY} \
```

Comment:
This sets the wrong var BTW

Reply:
Oops, good catch! Fixed in https://github.com/suborbital/sat/pull/42/commits/a5be2cedd25f166cd3c84038d4eeeb031319de2c